README.md

```diff
@@ -100,7 +107,7 @@ Run the *Illumina* pipeline with the following command:
 nextflow run phac-nml/ncov-dehoster -profile conda --illumina --directory <path/to/paired_reads/dir> --human_ref <path/to/reference>
 ```
 
-Minimum computational specs required for the Illumina dehosting pipeline are 3+ cpu and 16+G memory.
+Minimum computational specs required for the Illumina dehosting pipeline are 3+ cpu and 16+G memory. More resources may be needed depending on the size of the input fastq files.
```

Comment:
Is there any reference to an actual file-size threshold you can add here?

Reply:
I don't have any specific benchmarking statistics at the moment for cpus / mem required based on fastq size. That would be a good plot to add in later when I have a bit of time to do some benchmarking

The minimum ones should allow most files to run based on my little testing, just that if you have a 500 million reads as an input, using 2 cpus, and 16G mem it will take a lot of time